packages/luna/src/components/Loading/LoadingError.tsx

```diff
@@ -4,15 +4,24 @@ import { useFormatMessage } from "@/hooks/intl";
 import { Typography } from "@litespace/luna/Typography";
 import { Void } from "@litespace/types";
 import React from "react";
+import cn from "classnames";
 
-export const LoadingError: React.FC<{ retry: Void; error: string }> = ({
-  retry,
-  error,
-}) => {
+export const LoadingError: React.FC<{
+  retry: Void;
+  error: string;
+  variant?: "small" | "medium" | "large";
```

Comment:
Awesome work. This solution is really good and flawless. However, I have a suggestion: what if we defined the type of `variant` as follows `"10" | "16" | "20"`, and then embedded it in tailwind class name `tw-w-${variant}`. I think It could make the code more succinct.

Reply:
This will not work due to how tailwind scan for classnames.

Reply:
> This will not work due to how tailwind scan for classnames.

I think It will work fine, at least inside the `cn` utility function.